### DISCUSSION TITLE
Minimum HW Requirements (number of machines, CPUs, RAM) for a k8ssandra and JanusGraph local cluster with Ubuntu 22.04 as common O.S.?

### DISCUSSION THREAD

**Discussion by raphael10-collab** [JanusGraph/janusgraph#3637] [Q&A]
I looked around but I didn't fine any information...

What are the Minimum HW Requirements (number of machines, CPUs, RAM) for a k8ssandra and JanusGraph local cluster with Ubuntu 22.04 as common O.S.?

**Comment by hadoopmarc** [comment]
See: https://docs.janusgraph.org/operations/deployment/  Machines need not be big (4Gb, 2 cores) to start with, it all depends on your load. You also need some means for load balancing, virtual IP to create a single point of entry for multiple janusgraph servers.

**Comment by raphael10-collab** [comment]
Thank you very much


Il mer 1 mar 2023, 16:15 hadoopmarc ***@***.***> ha scritto:
